### ISSUE TITLE
Unhandled Promise Rejection Error: New version 1.16.0 is not signed by the application owner

### ISSUE THREAD

**Issue by jy95** [buttercup/buttercup-desktop#779]
```
Unhandled Promise Rejection
Error: New version 1.16.0 is not signed by the application owner: publisherNames: MadDev Oy, raw info: {
  "SignerCertificate": null,
  "TimeStamperCertificate": null,
  "Status": 2,
  "StatusMessage": "The file C://Users//jy95//AppData//Local//Temp//up-5x35Tw//buttercup-desktop-setup-1.16.0.exe is not digitally signed. You cannot run this script on the current system. For more information about running scripts and setting execution policy, see about_Execution_Policies at https:/go.microsoft.com/fwlink/?LinkID=135170"
}
    at t.newError (C:/Program Files/Buttercup/resources/app.asar/dist/app.js:1:10115)
    at C:/Program Files/Buttercup/resources/app.asar/dist/app.js:207:213461
    at Generator.next (<anonymous>)
    at Generator.c (C:/Program Files/Buttercup/resources/app.asar/dist/app.js:1:1411)
    at d._promiseFulfilled (C:/Program Files/Buttercup/resources/app.asar/dist/app.js:206:182460)
    at j._settlePromise (C:/Program Files/Buttercup/resources/app.asar/dist/app.js:206:132961)
    at j._settlePromise0 (C:/Program Files/Buttercup/resources/app.asar/dist/app.js:206:133559)
    at j._settlePromises (C:/Program Files/Buttercup/resources/app.asar/dist/app.js:206:134886)
    at j._fulfill (C:/Program Files/Buttercup/resources/app.asar/dist/app.js:206:133929)
    at j._settlePromise (C:/Program Files/Buttercup/resources/app.asar/dist/app.js:206:133058)
    at j._settlePromise0 (C:/Program Files/Buttercup/resources/app.asar/dist/app.js:206:133559)
    at j._settlePromises (C:/Program Files/Buttercup/resources/app.asar/dist/app.js:206:134886)
    at j._fulfill (C:/Program Files/Buttercup/resources/app.asar/dist/app.js:206:133929)
    at C:/Program Files/Buttercup/resources/app.asar/dist/app.js:96:4568
    at a (C:/Program Files/Buttercup/resources/app.asar/dist/app.js:206:216760)
From previous event:
    at j.B [as _captureStackTrace] (C:/Program Files/Buttercup/resources/app.asar/dist/app.js:206:151655)
    at new d (C:/Program Files/Buttercup/resources/app.asar/dist/app.js:206:181335)
    at C:/Program Files/Buttercup/resources/app.asar/dist/app.js:206:184424
    at t.NsisUpdater.downloadUpdate (C:/Program Files/Buttercup/resources/app.asar/dist/app.js:71:21210)
    at EventEmitter.i.ipcMain.on (C:/Program Files/Buttercup/resources/app.asar/dist/app.js:414:123679)
    at WebContents.<anonymous> (C:/Program Files/Buttercup/resources/electron.asar/browser/api/web-contents.js:284:13)
```

**Comment by sallar** [comment]
Thanks @jy95 I deleted the version so I can investigate without rush. Thanks for getting this to our attention.

**Comment by sallar** [comment]
@jy95 this should be fixed now, can you please re-download v1.16.0? Thanks
https://github.com/buttercup/buttercup-desktop/releases/tag/v1.16.0

**Comment by jy95** [comment]
Fixed on this version : thanks for your work
